src/tools/perf/perftest.c

```diff
@@ -324,11 +324,12 @@ static void usage(struct perftest_context *ctx, const char *program)
     printf("     -x <tl>        Transport to use for testing.\n");
     printf("     -c <cpu>       Set affinity to this CPU. (off)\n");
     printf("     -n <iters>     Number of iterations to run. (%ld)\n", ctx->params.max_iter);
-    printf("     -s <size>      Message size. (%zu)\n", ctx->params.message_size);
+    printf("     -s <size>      Message size. default is (%zu). It can contains (applicable in zcopy data layout only) a list of message sizes, separated by comma. For example, \"-s 16,48,8192,8192,14\"\n", ctx->params.message_size);
```

Comment:
1. "List of buffer sizes separated by comma, which make up a single message. Default is (%zu). "
1. limit any single line of code to 80 chars.